docs/sidebar.yml

```diff
@@ -81,6 +81,7 @@ subsection:
           - page: reference/other-new-features/safe-initialization.md
           - page: reference/other-new-features/type-test.md
           - page: reference/other-new-features/experimental-defs.md
+          - page: reference/other-new-features/binary-literals.md
```

Comment:
and there's another sidebar for the "stable reference" 😅

Reply:
Surprised to see the PR still open, as it's just 0's and 1's, but it's the dread "long tail of documentation".

Reply:
Okay, I'll fix that as well !